.github/workflows/django.yml

```diff
@@ -0,0 +1,56 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    notifications:
+      slack: djangogirls:f5ZWSNUTwKp1C8L8rr3PYjg7
+    cache:
+      directories:
+        - $HOME/.cache/pip
+
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: djangogirls
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
Do we need all these versions? 🤔

Reply:
We are using 3.4 in production, was using 3.5 in Travis and we might want to move to Django 2.2 or higher and a higher version of Python. How do we handle all those variations?

Reply:
I'd suggest one at a time... 

It's ok to run all these. Let's see if it's compatible, then we can drop the older ones as we update.